docs/source/AdministratorGuide/Systems/Transformation/index.rst

```diff
@@ -589,6 +607,12 @@ Generation of bulk data removal/replication requests from a fixed file list or a
     transID = t.getTransformationID()
     tc.addFilesToTransformation(transID['Value'],infileList) # Files are added here
 
+**Note:**
+
+  * This body will create Requests with 2 operations each. The first element of the tuple is the operation type, the second is a dictionary describing the attributes of the Operation
+  * If a value of an attribute is ``TASK:<something>``, this will resolve into the ``<something>`` attribute of the task dictionary, typically the target SE.
```

Comment:
I would welcome a concrete example here, in the administrator guide, rather than in the developer guide.

Reply:
the example is above: https://github.com/DIRACGrid/DIRAC/pull/5547/files#diff-1d10beaf19bab7671040bdf28084ec5db0528ec96f035ced8dcbcc9cd419ee83R599

Reply:
What I mean is that I would move the sentence starting with "Imagine a world where " that is in bodyPlugin.rst here.